src/core/unicode.js

```diff
@@ -1630,8 +1630,41 @@ function reverseIfRtl(chars) {
   return s;
 }
 
+function normalizeUnicode(str, normalizedUnicodesMap) {
+  if (str.length === 1) {
+    if (normalizedUnicodesMap[str] !== undefined) {
+      return normalizedUnicodesMap[str];
+    }
+    return str;
+  }
+
+  const unicodeBuf = [];
+  for (let i = 0, ii = str.length; i < ii; i++) {
+    const unicode = str[i];
+    const normalized = normalizedUnicodesMap[unicode];
+    if (normalized === undefined) {
+      // If a toUnicode entry contains two same unified Unicode
+      if (unicodeBuf.includes(unicode) === false) {
```

Comment:
Instead of `=== false`, use `!unicodeBuf.includes(unicode)` instead.

Reply:
This entire check is definitely not correct, and it should thus explain the test failures seen with this patch.

Generally speaking, the regression clearly outlines that there's no reason why a ToUnicode entry cannot contain the same glyph more than once.